src/main/kotlin/com/terraformation/backend/tracking/api/PlantingSitesController.kt

```diff
@@ -359,7 +366,9 @@ data class PlantingZoneReportedPlantsPayload(
     val plantsSinceLastObservation: Int,
     val plantingSubzones: List<PlantingSubzoneReportedPlantsPayload>,
     val progressPercent: Int,
+    val species: List<ReportedSpeciesPayload>,
```

Comment:
Should `species` be a set instead of a list? Unsure if it's already filtered out duplicates

Reply:
Looking at the queries, I don't think there's a way there could be duplicates.